src/sandbox/runtime/actions/edit.js

```diff
@@ -55,3 +70,20 @@ export const clearRange = undoable((thisSheet, range) => {
   const found = query(thisSheet.get().blocks).getInRange(normalize(range));
   for (let blockId of found) clearBlock(thisSheet, blockId);
 });
+
+export const commentRange = undoable((thisSheet, range, cb) => {
```

Comment:
It's very unclear what cb is for. Can it be anything but just toggling "//" ?

Reply:
I agree. There's actually no other possible functionality here except toggling comments.
Fixed.